lib/build-view.js

```diff
@@ -131,12 +132,15 @@ export default class BuildView extends View {
   }
 
   resizeToNearestRow() {
+    if ( typeof this.cache === 'undefined' ) return;
```

Comment:
This confused me too... was getting an error because resizeToNearestRow was being called before cache was initialised in constructor. How?! Had a chance to look into this today and turns out, atom.config.observe('editor.fontSize') returns immediately with the current fontSize, which then calls resizeToNearestRow through fontSizeFromConfig. I've moved the cache init ahead of the config subscriptions, and removed the undefined check.